Tests/Marketplace/marketplace_services.py

```diff
@@ -1056,9 +1056,21 @@ def prepare_release_notes(self, index_folder_path, build_number):
         task_status = False
         not_updated_build = False
 
+        print('doing plain ls')
```

Comment:
is this ready for review?

Reply:
No, currently debugging how to use this solution to fix the index